ggmolvis/properties/colors.py

```diff
@@ -8,6 +8,10 @@
 class Color(Property):
     """Class for the color."""
 
+
+    def __init__(self, mol_obj, color):
+        self.colors = [color]
```

Comment:
Looking at the current control flow--this may need to be adjusted to have an entry for each "frame." That seems a bit awkward--I suppose maybe there is a desire to change color on a per-frame basies.